contributing-guides/maintainers-guide.md

```diff
@@ -85,18 +85,26 @@ as a guideline for current and future maintainers.
   or if the multiple commits are not semantically independent changes,
   use the `Squash and merge` method.
   (Don't forget to clean up the body of the squashed commit message.)
-  If instead the PR author took the time to craft
+  If instead, the PR author took the time to craft
   individual, informative messages for each commit,
   then use the `Rebase and merge` method,
   to honor that work and preserve the history of the changes.
   For less clear-cut cases, a simple heuristic you can follow
   is that if there are more "dirty" commits than "clean" commits,
   then prefer squash, else do a rebase.
 
-- Although having push access allows committing directly to the repository,
-  please **create pull requests for all of your changes**,
-  except the simplest ones (e.g. typo fixes).
+- Although having push access allows committing directly to the repository to all branches (except main),
+  please **create pull requests for all of your changes**.
   This ensures that the entire process that regular contributors go through
   is also exposed to maintainers,
   who can then identify and address bottlenecks or inconveniences.
   Similarly, **avoid merging your own PRs** unless approved by other maintainers.
+
+- At the last week of October, all applicable PRs that wouldn't get merged
+ in time can be labeled as `hacktoberfest-accepted`.
+
+## III. Transparency
+
+- All non-confidential requests/mail made/sent on behalf of the project
+ should be documented as an issue with the [archive](https://github.com/tldr-pages/tldr/labels/archive) label
```

Comment:
```suggestion
 should be documented as an issue with the [archive](https://github.com/tldr-pages/tldr/issues?q=label%3Aarchive) label
```

Perhaps this link is better? That way, it takes the reader to both open and closed issues, rather than just the open ones.